### ISSUE TITLE
DD Tech: Advanced Time Entry Adjusted End Time

### ISSUE THREAD

**Issue by dr3dgen** [DeskDirector/UserVoice#345]
In DD Tech, can we have an option that recognizes the time period (AM or PM) in the start time field and mirrors it in the end time field by default? I'm thinking that most issues probably don't happen overnight and would make for easier adjusted time entry. Right now, we're changing it manually for each ticket. It might seem small, but it can become tedious over time.

**Comment by Andrew-Lahikainen** [comment]
@dr3dgen So when you change the start time period from AM to PM (and vice versa), you want the end time period to automatically reflect that? Did I get that right?

Do you find that you need to use the advanced options quite often?

**Comment by dr3dgen** [comment]
Yes, precisely. That way it's a quicker time entry using the tab key across from start to finish time. Minimal mouse interaction is super helpful across functions, so more of that would be great!

I love the auto-generated time. It helps us to gauge how much time was spent when we start a ticket. Our SLAs currently round up to the nearest 15 minute interval, though, so we have to make minor adjustments on each entry.
